config/deploy/stage.rb

```diff
@@ -0,0 +1,6 @@
+ENV['APP_ENV']  ||= 'staging'
+require_relative 'server_settings'
+
+server 'sdr-services-test.stanford.edu', user: 'sdr2service', roles: %w{app}
+
+Capistrano::OneTimeKey.generate_one_time_key!
```

Comment:
The SDR-PC has both `stage` and `prod` systems; whenever we want to do any QA on the integrated systems, we run a bunch of stuff through accessioning on the `stage` systems and we expect them to mimick the `prod` systems for the QA to be a real integration test of all the DOR/SDR microservices.  (The sdr-services-app has various configs to deal with the fact that it can be developed 'local' or in 'test' using mysql, and also deployed to an `integration` system but the latter system might be still using oracle instead of mysql).  Yea.